Sources/Public/Animation/LottieAnimationViewInitializers.swift

```diff
@@ -23,7 +23,7 @@ extension LottieAnimationView {
     name: String,
     bundle: Bundle = Bundle.main,
     imageProvider: AnimationImageProvider? = nil,
-    animationCache: AnimationCacheProvider? = LRUAnimationCache.sharedCache,
+    animationCache: AnimationCacheProvider? = LottieConfiguration.shared.animationCache,
```

Comment:
I don't love that we both have an `animationCache` param and a `configuration` one and the `animationCache` isn't coming from the `configuration`.

However I couldn't think about any good ways to deal with this since a `nil` `animationCache` is supported.

Reply:
Hm, this is quite the gotcha. 

What if we created a separate `LottieAnimationCache.default` singleton that we use for this, instead of adding it to `LottieConfiguration`? 

That would have the added benefit of letting `LottieConfiguration` stay a simple value type with synthesized Equatable / Hashable conformances.

Reply:
Updated!